### DISCUSSION TITLE
Mustang validate Cli command

### DISCUSSION THREAD

**Discussion by amg211** [ZUGFeRD/mustangproject#386] [Q&A] [answered]
Hi,
I'm using the mustang cli Version to combine PDF and XMLs. The Combination is working fine but when I try to validate the Output PDF with the validate command it shows me an error that my PDF is invalid because of the validation via the PDF/A-3U validation profile.

The following error message is showing up:

> The Launch, Sound, Movie, ResetForm, ImportData, Hide, SetOCGState, Rendition, Trans, GoTo3DView and JavaScript actions shall not be permitted. Additionally, the deprecated set-state and no-op actions shall not be permitted.

I checked my input PDF and it's PDF/A-3A not PDF/A-3U but the converted PDF is indeed an PDF/A-3U. I thought for Zugferd you would need PDF/A-3A. Is that correct or do I have another option for validating a pdf except the --validate --source Command?

Thank you in advance! :)
Best regards 
Anna-Maria

**Comment by jstaerk** [comment, marked as answer]
ZUGFeRD indeed requires PDF/A-3 (or PDF/A-4, interestingly) (and Mustang --combine usually uses PDF/A-1 as input) but it doesn't matter if it's A-3 A, B or U. Mustang uses a embedded verapdf to check validity, it's open source, you can download and run that separately as well. To correct a PDF/A (losing ZF info) I recommend just running it through Ghostscript PDF/A conversion, if you really need to deep-dive to fix PDF issues on the almost-binary level I suggest itext RUPS.

**Comment by amg211** [comment]
Thank you! I figured out, that my PDF Printer made an automatic conversion of mails or website addresses to URIs which is not allowed in PDF/A-3!
